modules/yarn.lock

```diff
@@ -16238,7 +16238,24 @@ send@0.17.1:
     range-parser "~1.2.1"
     statuses "~1.5.0"
 
-senna@2.7.8, senna@^2.3.1:
```

Comment:
Hey @brunobasto, is there a reason for this `yarn.lock` change? How did it creep in here?

We had to revert the `3.x.x` version of senna because we found severe regressions with redirections, so we're postponing the integration until after SP1.

Can you check if this was caused by a rebase or a weird dependency and resend?

Reply:
In fact, this is just a duplicate, but totally unnecessary and will just add bloat to the download... you can run `yarn why senna` to see why did it make it here, which is totally unexpected :/

Reply:
No reason. To be honest I didn't even look. I assumed the change was because of clay-sticker. Which would make sense since I added that to my package json. Will resend removing that. Thx

Reply:
Will try to be more careful next time :)

Reply:
This branch is like a week old, so it might have been bad conflict resolution on my end. Thx for the yarn why tip. Didn't know abou that!